easybuild/easyconfigs/c/Circlator/Circlator-1.5.5-foss-2023a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'Circlator'
+version = '1.5.5'
+
+homepage = 'https://github.com/sanger-pathogens/circlator/'
+description = """A tool to circularize genome assemblies.s"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('openpyxl', '3.1.2'),
+    ('Fastaq', '3.17.0'),
+    ('Pysam', '0.22.0'),
+    ('MUMmer', '4.0.0rc1'),
+    ('BWA', '0.7.17'),
+    ('prodigal', '2.6.3'),
+    ('SPAdes', '3.15.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pymummer', '0.11.0', {
+        'checksums': ['199b8391348ff83760e9f63fdcee6208f8aa29da6506ee1654f1933e60665259'],
+    }),
+    (name, version, {
+        'source_urls': [
+            'https://files.pythonhosted.org/packages/7c/bf/f0fd534a733b7d98fc9853ca1aa735984cec1e5fc4670b632e76973f8add/'
```

Comment:
You can avoid having to specify `source_urls` at all by using lowercase name as extension:

```python
('circulator', version, {
```

`PythonBundle` by default knows that extensions should be downloaded via PyPI, you don't need to specify that yourself